CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- update the client fallbacks to caching for 5 minutes if Cache-Control is missing or malformed.
```

Comment:
"update" should not be used in a changelog, which by definition is a list of updates, in the broad sense. Also, each item should be self contained so in the 2nd line, "that value" is unclear.

How about in one sentence:

> Cache responses for 5 mins (non configurable) as a fallback when the Cache-Control header is missing or malformed.

Reply:
updated as suggested in https://github.com/mdsol/mauth-jvm-clients/pull/136/commits/0a20b9d96eefba7140a4024e8e64e3309ff0a67a